src/Chainweb/Utils/Paging.hs

```diff
@@ -129,6 +132,15 @@ data NextItem k
     | Exclusive k
     deriving stock (Eq, Show, Ord, Functor, Foldable, Traversable)
 
+_getNextItem :: NextItem k -> k
+_getNextItem (Inclusive k) = k
+_getNextItem (Exclusive k) = k
+{-# INLINE _getNextItem #-}
+
+getNextItem :: Getter (NextItem k) k
+getNextItem = to _getNextItem
```

Comment:
This throws a redundant constraint warning.

Reply:
Yes. Didn't the team agree sometime ago to remove the redundant-constraint warning flag from the cabal file?

Reply:
I guess removing it from the cabal file should be in a separate PR. So I'll add the the pragma to suppress it.

Reply:
accidentally committed this into #163. So, won't fix it in the PR.